lineman/app/components/navbar/navbar_user_options.coffee

```diff
@@ -3,5 +3,9 @@ angular.module('loomioApp').directive 'navbarUserOptions', ->
   restrict: 'E'
   templateUrl: 'generated/components/navbar/navbar_user_options.html'
   replace: true
-  controller: ($scope, CurrentUser) ->
+  controller: ($scope, CurrentUser, Records, $http) ->
```

Comment:
A SessionRecordsInterface _might_ not be necessary (although it might if we decide to add log in to this thing)

But for now, I reckon something like

```
@sessionClient = new RestfulClient('sessions')
@sessionClient.destroy().then -> ...
```

would do us for now.